modules/oe_theme_content_publication/oe_theme_content_publication.post_update.php

```diff
@@ -65,10 +66,49 @@ function oe_theme_content_publication_post_update_00004(): void {
   $display->save();
 }
 
+/**
+ * Enable Field Group and OpenEuropa Theme Content Entity Contact modules.
+ */
+function oe_theme_content_publication_post_update_00005(): void {
+  \Drupal::service('module_installer')->install(['oe_theme_content_entity_contact']);
```

Comment:
Contrary to what the comment says this only enabled one module.

Reply:
Fixed